docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.9"  # optional since v1.27.0
+services:
+  bot:
+    build: .
+    env_file: .env
+    environment:
+      REDIS_URL: "redis://redis:6379/1"
+    links:
+      - redis
```

Comment:
Is links required here? By default the containers will be able to access others, [links is for defining aliases afaics](https://docs.docker.com/compose/networking/#links).